rosdep/python.yaml

```diff
@@ -2216,6 +2216,37 @@ python-hypothesis:
     xenial_python3: [python3-hypothesis]
     yakkety_python3: [python3-hypothesis]
     zesty_python3: [python3-hypothesis]
+python-igraph-pip: &migrate_eol_2025_04_30_python3_igraph_pip
+  alpine:
+    pip:
+      packages: [python-igraph]
+  arch:
+    pip:
+      packages: [python-igraph]
+  debian:
+    pip:
+      packages: [python-igraph]
+  fedora:
+    pip:
+      packages: [python-igraph]
+  gentoo:
+    pip:
+      packages: [python-igraph]
```

Comment:
Pip can't be used on Gentoo according to the CONTRIBUTING guide. Mind deleting this?

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#gentoo

Reply:
Removed